# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../libebml/stage9.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2006-03-30 21:05:11.000000000 +0200
+++ ./Makefile	2006-09-28 22:24:50.000000000 +0200
@@ -141,7 +141,7 @@
 install_sharedlib: $(LIBRARY_SO)
 	$(INSTALL) $(INSTALL_DIR_OPTS) -d $(libdir)
 	$(INSTALL) $(INSTALL_OPTS_LIB) $(LIBRARY_SO_VER) $(libdir)
-	ln -s $(LIBRARY_SO_VER) $(libdir)/$(LIBRARY_SO)
+	ln -sf $(LIBRARY_SO_VER) $(libdir)/$(LIBRARY_SO)
 
 
 ifneq ($(wildcard .depend),)
